### ISSUE TITLE
Include graph for two systemd runs (initramfs and normal systemd)

### ISSUE THREAD

**Issue by paulmenzel** [systemd/systemd-bootchart#42]
Using [dracut](https://dracut.wiki.kernel.org/index.php/Main_Page) as the framework for the initrd, systemd is used in the initrd image, and systemd-bootchart only seems to shows that execution, and not the second one.

**Comment by jamuir** [comment]
this issue is almost 7 years old now.

does the defect still exist?

can you provide steps to reproduce?  (e.g. with a Fedora image)

**Comment by sofar** [comment]
Yeah this isn't easy to implement anyway, and I would argue that it's a whole lot simpler to just run a separate bootchart in each enviroment - i.e. run one bootchart in the initramfs, and a second one inside the normal boot enviroment.
